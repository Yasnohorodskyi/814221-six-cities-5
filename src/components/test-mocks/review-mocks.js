export const review = {
  comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
  date: `2019-05-08T14:13:56.569Z`,
  id: 1,
  rating: 4,
  user: {
    photo: `/img/avatar-angelina.jpg`,
    id: 4,
    isPro: false,
    name: `Lena`,
  },
};

export const reviews = [
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:53.569Z`,
    id: 1,
    rating: 4,
    user: {
      photo: `/img/avatar-angelina.jpg`,
      id: 4,
      isPro: false,
      name: `Lena`,
    },
  },

  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:56.569Z`,
    id: 2,
    rating: 5,
    user: {
      photo: `/img/avatar-angelina.jpg`,
      id: 34,
      isPro: true,
      name: `Masha`,
    },
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:58.569Z`,
    id: 3,
    rating: 3,
    user: {
      photo: `/img/avatar-angelina.jpg`,
      id: 39,
      isPro: false,
      name: `Masha`,
    },
  },
];

export const reviewsBeforeAdapt = [
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:56.569Z`,
    id: 2,
    rating: 5,
    user: {
      avatar_url: `/img/avatar-angelina.jpg`,
      id: 34,
      is_pro: true,
      name: `Masha`,
    },
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:58.569Z`,
    id: 3,
    rating: 3,
    user: {
      avatar_url: `/img/avatar-angelina.jpg`,
      id: 39,
      is_pro: false,
      name: `Masha`,
    },
  },
];
