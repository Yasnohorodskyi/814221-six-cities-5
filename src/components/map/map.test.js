import React from "react";
import renderer from "react-test-renderer";
import Map from "../map/map";
import {offers} from "../test-mocks/offers-mocks";

describe(`<Map /> render`, () => {
  const div = document.createElement(`div`);
  div.id = `map`;
  document.body.appendChild(div);
  it(`renders correctly with full information`, () => {
    const mapComponent = renderer
      .create(<Map offers={offers} activeItem={5} />)
      .toJSON();

    expect(mapComponent).toMatchSnapshot();
  });
});
