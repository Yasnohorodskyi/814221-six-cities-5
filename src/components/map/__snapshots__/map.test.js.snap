// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map /> render renders correctly with full information 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;
